### DISCUSSION TITLE
Bluetooth/USB-C to RS232/485 for iPad Air5/pro and iPhone 15

### DISCUSSION THREAD

**Discussion by Smiril** [isontheline/pro.webssh.net#1138] [Ideas]
It would become a special touch when WebSSH pro
Can speak over Bluetooth and usb-c to rs232/485 ports

the console serial port on a mobile phone or tablet via usb-c to rs232/485 adapter could become a new age of audio engineering 

look at this product [here](https://www.thomann.de/at/prod_AR_435191.html) .
This even makes a better idea for what and why to program 

why is this option not available? The rs232 port console

**Comment by isontheline** [comment]
Hello @Smiril 👋

Thanks for your issue 🙏

As WebSSH is inside a sandbox I can't access all peripherals.

Could you share some bluetooth dongles which work with your iPhone?

**Comment by Smiril** [comment]
that`s sad ... I`ve not tried any bluetooth dongle 
expect onboard bluetooth with raspberry pi for streaming music
forgive me for that idea but is it possible to access usb stick from app 
so I think it can be possible to reach the usb-c port from sandbox
on Ipad Air5  I can access the usb-stick in some apps ... must be possible to make data connection

even I think

**Reply by isontheline** [reply]
Which USB stick are you using?

**Reply by Smiril** [reply]
standard once like sandisk 16GB or else FAT32 formatted
